.gitignore

```diff
@@ -0,0 +1,109 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This gitignore file is not needed. Its in the main directory. The UI guys have their own gitignore for node